### PR TITLE
added first pass at release notes

### PR DESCRIPTION
**Pull request by alexwykoff**
Auditors: @bbondy , @luixxiul

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
 # Changelog
 
 ## [0.12.6](https://github.com/brave/browser-laptop/releases/v0.12.6dev)
-- (TODO)
+- Added the ability to set Brave as your default browser ([#4937](https://github.com/brave/browser-laptop/issues/4937))([#4958](https://github.com/brave/browser-laptop/issues/4958))([#4935](https://github.com/brave/browser-laptop/issues/4935))([#4934](https://github.com/brave/browser-laptop/issues/4934))([#2105](https://github.com/brave/browser-laptop/issues/2105))([#4939](https://github.com/brave/browser-laptop/issues/4939))
```
Comment (bbondy) [inline CHANGELOG.md]: Please add 
on startup and on the preferences page.

Comment (bbondy) [inline CHANGELOG.md]: Also these should all have a period at the end of each sentence please

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+- Added the option to opt-in to reporting usage stats ([#4691](https://github.com/brave/browser-laptop/issues/4691))
+- Fixed various security issues ([#4747](https://github.com/brave/browser-laptop/issues/4747))([#4885](https://github.com/brave/browser-laptop/issues/4885))([#4913](https://github.com/brave/browser-laptop/issues/4913))([#4883](https://github.com/brave/browser-laptop/issues/4883))
+- Fixed error on invalid homepage URL ([#4974](https://github.com/brave/browser-laptop/issues/4974))
+- Fixed Autocomplete to behave more like Chrome / Firefox ([#3049](https://github.com/brave/browser-laptop/issues/3049))([#3235](https://github.com/brave/browser-laptop/issues/3235))([#4861](https://github.com/brave/browser-laptop/issues/4861))
```
Comment (bbondy) [inline CHANGELOG.md]: Please avoid direct comparisons to other browsers, and just say Fixed autocomplete handling so it works better.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+- Fixed Favicons not being loaded when bookmarks imported from browser ([#4882](https://github.com/brave/browser-laptop/issues/4882))
```
Comment (bbondy) [inline CHANGELOG.md]: ...from other browsers.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,21 @@
...
+- Fixed double-clicking of entries on about:history opening a blank new tab ([#4909](https://github.com/brave/browser-laptop/issues/4909))
+- Fixed Pressing Tab, then Enter in URL bar unexpectedly minimizing window on Windows ([#4846](https://github.com/brave/browser-laptop/issues/4846))
```
Comment (bbondy) [inline CHANGELOG.md]: in *the* URL bar

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,7 +1,21 @@
...
+- Fixed address bar suggestions not working if you are offline ([#3730](https://github.com/brave/browser-laptop/issues/3730))
```
Comment (bbondy) [inline CHANGELOG.md]: Address bar or URL bar but be consistent.

**Comment by alexwykoff** [comment]
updated release notes with suggestions

**Comment by bbondy** [comment]
++

**Comment by luixxiul** [comment]
++

**Pull request merged**